src/locales/de-DE.json

```
@@ -153,6 +153,11 @@
       },
       "unknownOrganisation": "Unbekannte Organisation"
     },
+    "addPerson": {
+      "management":"Person hinzufügen",
+      "searchStateEmployee":"Landesbediensteter (Nur Suche)",
+      "createNew":"Andere Person (Neuanlage)"
+    },
```

I'd put this as a new block under the "person": block 

```suggestion
    "adminSearchsForTeacher": {
      "addPerson":"Person hinzufügen",
      "searchStateEmployee":"Landesbediensteter (Nur Suche)",
      "anotherPerson":"Andere Person (Neuanlage)"
    },
```